src/patternfly/components/Form/examples/Form.md

```diff
@@ -13,7 +13,7 @@ cssPrefix: pf-c-form
       {{#> form-label form-label--attribute=(concat 'for="' form--id form-group--id '"') required="true"}}
         Name
       {{/form-label}}
-      {{> form-group-label-help}}
+      {{> form-group-label-help form-group-label-help--aria-label="More information for name field" form-group-label-help--aria-describedby=(concat form--id form-group--id)}}
```

Comment:
Looks like https://github.com/patternfly/patternfly/pull/4714/files#diff-604e9154a4176f06e862bcb8493641aea5f98be03d3ea0e9935ca8935b229fdaR11 needs to be something like `{{#> form-group form-group--id=(concat form--id "-name")}}` or added to `form-group.hbs`